apps/services/open_feature/providers/dev_cycle/feature_flags.py

```
@@ -0,0 +1,257 @@
+                    default_value=default_value,
+                    evaluation_context=context
+                )
+                
+                # Check if we got a valid result (not the default)
+                if result != default_value:
+                    logger.debug(f"Provider variable {variable_key}: {result}")
+                    return result
+                else:
+                    logger.debug(f"Provider variable {variable_key} not found or default, using environment fallback")
+                    
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)